packages/web/src/pages/upload-page/fields/RemixSettingsField/RemixSettingsMenuFields.tsx

```diff
@@ -34,14 +35,16 @@ const messages = {
 
 export const RemixSettingsMenuFields = () => {
   const [{ value: trackUrl }] = useField(REMIX_LINK)
-  const permalink = getPathFromTrackUrl(trackUrl)
+  const permalink = useThrottle(getPathFromTrackUrl(trackUrl), 1000)
```

Comment:
note @amendelsohn this is pretty sick. Before we would fetch on every change to trackUrl, now it has a search functionality. we could further optimize by setting disabled: true until trackUrl actually matches a real url.